content/en/developers/metrics/custom_metrics.md

```diff
@@ -3,226 +3,244 @@ title: Custom Metrics
-After submitting a distribution metric to Datadog, you have the option to add percentile aggregations to a distribution with the Distributions UI in-app. Distributions with percentile aggregations are counted differently compared to the metric types listed above since percentiles are not mathematically reaggregatable.
 
-Suppose you are interested in measuring the *median* `age` in the state of New York where the `age` distribution metric is tagged with `city` and `state`. 
+After submitting a distribution metric to Datadog, you have the option to add percentile aggregations to a distribution with the [Distributions UI][11]. Distributions with percentile aggregations are counted differently compared to the metric types listed above since percentiles are not mathematically reaggregatable.
+
+Suppose you are interested in measuring the *median* `age` in the state of New York where the `age` distribution metric is tagged with `city` and `state`.
 
-|  | Values in 10s flush interval | Sum | Count | Min | Max | Avg | p50 | p75 | p90 | p95 | p99 |
+| Tags          | Values in 10s flush interval | Sum | Count | Min | Max | Avg | p50 | p75 | p90 | p95 | p99 |
 |---------------|------------------------------|-----|-------|-----|-----|-----|-----|-----|-----|-----|-----|
-| Rochester, NY | 23,33,55,41,36,12,66 | 266 | 7 | 12 | 66 | 38 | 23 | 55 | 66 | 66 | 66 |
-| New York, NY | 18,26,31,29,40,23,36 | 203 | 7 | 18 | 40 | 29 | 29 | 36 | 40 | 40 | 40 | 
+| Rochester, NY | 23,33,55,41,36,12,66         | 266 | 7     | 12  | 66  | 38  | 36  | 55  | 66  | 66  | 66  |
+| New York, NY  | 18,26,31,29,40,23,36         | 203 | 7     | 18  | 40  | 29  | 29  | 36  | 40  | 40  | 40  |
 
-Percentiles are NOT reaggregatable -- you can't reaggregate the same way maximum ages were above. The median age in New York is not equal to the `median`(`median`(Rochester, NY), `median`(New York, NY)). 
+Percentiles are NOT reaggregatable—you can't reaggregate the same way maximum ages were above. The median age in New York is not equal to the `median`(`median`(Rochester, NY), `median`(New York, NY)).
 
-Therefore, Datadog needs to precalculate five timeseries (`p50`,`p75`,`p90`,`p95`,`p99`) for each potentially queryable tag value combination. In the New York example, you have the following potentially queryable tag value combinations: 
- * Rochester, (`null` state)
- * New York, (`null` state)
- * (`Null` city), NY
- * Rochester, NY
- * New York, NY
- * (`Null` city), (`null` state) -- equivalent to * {all time series}
+Therefore, Datadog needs to precalculate five timeseries (`p50`,`p75`,`p90`,`p95`,`p99`) for each potentially queryable tag value combination. In the New York example, you have the following potentially queryable tag value combinations for the `city` and `state` tags:
 
-There are three potentially queryable values for the `city` tag: {Rochester, New York, `null`} and two values for the `state` tag: {NY, `null`}.
+| city tag  | state tag |
```

Comment:
I'm not sure `null` would make sense to a user. It was really to express that you can have a timeseries with just Rochester and no specification of state tag. Maybe replacing null with empty ? or empty (equivalent to null) would get that point across